iFixit/product-page-components
# PPC: Add store and language dropdowns

This is mostly a straight copy-paste from our main repo (after https://github.com/iFixit/ifixit/pull/35955 is merged). However, there are a few differences due to differences in the configuration of this repo vs. our main one.

![image](https://user-images.githubusercontent.com/51836911/105426034-49549480-5bff-11eb-8273-854125b8ed68.png)

## CR
Commit by commit because I try to explain what wasn't a straight copy-paste.

## QA
Run `npm start` and check that the language/store dropdowns look and function correctly. This QA doesn't need to be very thorough. We can do a more thorough QA when we use this version in our main repo. We want to get this out quickly so inkofpixel can use it in their footer redesign.

Note: I stubbed the `onClick` of the store dropdown links, so all clicking them does is print the store to the console.

CC @danielbeardsley @kwiens @sterlinghirsh

Connects https://github.com/iFixit/ifixit/issues/35956
